.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -4,27 +4,36 @@ about: Report a bug to help us improve cert-manager
 
 ---
 
-> Bugs should be filed for issues encountered whilst operating cert-manager.
-> You should first attempt to resolve your issues through the community support
-> channels, e.g. Slack, in order to rule out individual configuration errors.
-> Please provide as much detail as possible.
+<!--
+Bugs should be filed for issues encountered whilst operating cert-manager.
+You should first attempt to resolve your issues through the community support
+channels, e.g. Slack, in order to rule out individual configuration errors.
+Please provide as much detail as possible. 
+-->
 
 **Describe the bug**:
-A clear and concise description of what the bug is.
+<!--
+A clear and concise description of what the bug is. 
+Tip: you can use 
+```
+<code here>
+```
+for code blocks of your kubectl output or YAML files.
+-->
 
 **Expected behaviour**:
-A concise description of what you expected to happen.
+<!--A concise description of what you expected to happen.-->
 
 **Steps to reproduce the bug**:
-Steps to reproduce the bug should be clear and easily reproducible to help people
-gain an understanding of the problem.
+<!--Steps to reproduce the bug should be clear and easily reproducible to help people
+gain an understanding of the problem.-->
 
 **Anything else we need to know?**:
 
 **Environment details:**:
-- Kubernetes version (e.g. v1.10.2):
-- Cloud-provider/provisioner (e.g. GKE, kops AWS, etc):
-- cert-manager version (e.g. v0.4.0):
-- Install method (e.g. helm or static manifests):
+- Kubernetes version:
```

Comment:
Why is this more clear? I see a lot of issue filling in the actual value in the brackets which is very confusing to read expecting the examples